semapv-properties.tsv

```diff
@@ -0,0 +1,14 @@
+IRI	skos:prefLabel	skos:definition	dc:source	skos:example	rdfs:comment	altLabel	Parent	rdfs:isDefinedBy
+ID	A skos:prefLabel	A skos:definition	A dc:source	A skos:example	A rdfs:comment	A skos:altLabel	SC % SPLIT=|	A rdfs:isDefinedBy
+skos:mappingRelation	mapping relation	A mapping (alignment) link between two concepts.	https://www.w3.org/TR/skos-reference/					http://www.w3.org/2004/02/skos/core#
+skos:closeMatch	close match	A match where the subject and object are sufficiently similar that they can be used interchangeably in some information retrieval applications.	https://www.w3.org/TR/skos-reference/				skos:mappingRelation	http://www.w3.org/2004/02/skos/core#
+skos:exactMatch	exact match	A match where the subject and object are sufficiently similar, with a high degree of confidence, that they can be used interchangeably across a wide range of information retrieval applications.	https://www.w3.org/TR/skos-reference/				skos:closeMatch|semapv:isomorphicMatch	http://www.w3.org/2004/02/skos/core#
+skos:broadMatch	broad match	A match where the object is a broader concept than the subject.	https://www.w3.org/TR/skos-reference/				semapv:nonIsomorphicMatch	http://www.w3.org/2004/02/skos/core#
+skos:narrowMatch	narrow match	A match where the object is a narrower concept that the subject.	https://www.w3.org/TR/skos-reference/				semapv:nonIsomorphicMatch	http://www.w3.org/2004/02/skos/core#
+skos:relatedMatch	related match	A match where the subject and the object are inherently related, but where none is in any way more general than the other.	https://www.w3.org/TR/skos-reference/				semapv:nonIsomorphicMatch	http://www.w3.org/2004/02/skos/core#
+semapv:isomorphicMatch	isomorphic match	A match where the subject is isomorphic to the object, i.e. considered of identical or similar form, shape, or structure.	https://github.com/mapping-commons/semantic-mapping-vocabulary/issues/3				skos:mappingRelation	
+semapv:nonIsomorphicMatch	non-isomorphic match	A match where the subject cannot be considered isomorphic to the object, i.e. considered of identical or similar form, shape, or structure. The object corresponds to exactly one subject in the subject_source.	https://github.com/mapping-commons/semantic-mapping-vocabulary/issues/3				skos:mappingRelation	
+semapv:crossSpeciesExactMatch	cross-species exact match	A match where the subject is considered analogous to the object in a different taxonomic grouping.	https://github.com/mapping-commons/semantic-mapping-vocabulary/issues/3	FBbt:00005074 (D. melanogaster “muscle cell”) and CL:0000187 (taxon-neutral “muscle cell”) are a cross-species exact match.			semapv:isomorphicMatch|skos:relatedMatch	
```

Comment:
```suggestion
semapv:crossSpeciesExactMatch	cross-species exact match	A match where the subject is considered analogous to the object in a different taxonomic grouping.	https://github.com/mapping-commons/semantic-mapping-vocabulary/issues/3	FBbt:00005074 (D. melanogaster "muscle cell") and CL:0000187 (taxon-neutral "muscle cell") are a cross-species exact match.			semapv:isomorphicMatch|skos:relatedMatch	
```

Reply:
Ah, someone doesn’t like typographically correct quote characters! :D Fine, I’ll change them – reluctantly, for the record.

Reply:
What do you mean typographically correct? Is the one you put the right one? If so you should ignore my comment!

Reply:
If you care about typography (most people don’t, and it’s probably fine – I am one of the annoying few who do :p ), in English you should never use straight quotes (`"`). They are a typographical abomination from the era of typewriters, allowed to persist in the computer age by the use of 7-bits character encodings.

Proper quotes should be _curved_, with the opening quote (`“`, U+201C) being different from the closing quote (`”`, U+201D).

Same for the apostrophe. A proper apostrophe is curved (`’`), _not_ straight (`'`).

That’s just a pet peeve of mine. Poor typography makes me sad. :p